js/web/lib/wasm/jsep/webgpu/ops/cumsum.ts

```diff
@@ -0,0 +1,78 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+import {DataType} from '../../../wasm-common';
+import {TensorView} from '../../tensor-view';
+import {ShapeUtil} from '../../util';
+import {AttributeWithCacheKey, createAttributeWithCacheKey} from '../attribute-with-cache-key';
+import {ComputeContext, ProgramInfo} from '../types';
+
+import {createTensorShapeVariables, inputVariable, outputVariable, ShaderHelper} from './common';
+
+
+export interface CumSumAttributes extends AttributeWithCacheKey {
+  readonly exclusive: number;
+  readonly reverse: number;
```

Comment:
use boolean

Reply:
Yeah. Will do. I thought about this at time of coding but I was not sure whether it makes any difference.